doc/howto/instances_console.md

```diff
@@ -0,0 +1,28 @@
+---
+discourse: 9223
+---
+
+(instances-console)=
+# How to access the console
+
+Use the `lxc console` command to attach to instance consoles.
+The console is available at boot time already, so you can use it to see boot messages and, if necessary, debug startup issues of a container or VM.
```

Comment:
We should also mention that one can access the console immediately at instance start time via `lxc start <instance> --console` or `--console=vga` for VMs.